versions.go

```diff
@@ -73,43 +86,90 @@ type LegacyMongoDBVersion struct {
 // NewMongoDBVersion is a structure representing a version identifier for versions of
 // MongoDB, which implements the MongoDBVersion.
 type NewMongoDBVersion struct {
-	LegacyMongoDBVersion
+	LegacyMongoDBVersion // note not all fields are applicable to NewMongoDBVersion
+	isDevRelease bool
+	devReleaseNumber int
+	quarter string
+}
+
+// Series is not applicable to new versions, so always return the empty string.
+func (v *NewMongoDBVersion) Series() string {
+	return ""
+}
+
+func (v *NewMongoDBVersion) Quarter() string {
+	return v.quarter
+}
+
+// IsStableSeries is not applicable to new versions, so always return false.
+func (v *NewMongoDBVersion) IsStableSeries() bool {
+	return false
+}
+
+// IsDevelopmentSeries is not applicable to new versions, so always return false.
+func (v *NewMongoDBVersion) IsDevelopmentSeries() bool {
+	return false
+}
+
+// IsInitialStableReleaseCandidate is not applicable to new versions, so always return false.
+func (v *NewMongoDBVersion) IsInitialStableReleaseCandidate() bool {
+	return false
+}
+
+// StableReleaseSeries is not applicable to new versions, so always return the empty string.
+func (v *NewMongoDBVersion) StableReleaseSeries() string {
+	return ""
+}
+
+// IsLTS returns true if this is the year
```

Comment:
if this is the first release of the year.

Reply:
oops haha